### DISCUSSION TITLE
Missing wslu package from version 3.2.2 assets

### DISCUSSION THREAD

**Discussion by tranzystorekk** [wslutilities/wslu#178] [Q&A] [answered]
As in the title, there is no `.zst` file in the 3.2.2 release assets, I'm not sure how to install it on Arch Linux

**Comment by car-meows** [comment, marked as answer]
Hi, 

All releases are gradually rolling out, being released doesn't mean they are immediately available for every distro. Packages for Arch Linux, Kali Linux, Ubuntu, and Alpine Linux all needed to be manually packaged. Hope you understand.

**Comment by car-meows** [comment]
The package is now live; You can download it from release.
